src/test/java/se/sundsvall/invoicesender/integration/db/DbIntegrationTests.java

```diff
@@ -54,6 +56,36 @@ void testGetBatches() {
 		verifyNoMoreInteractions(batchRepositoryMock);
 	}
 
+	@Test
+	void testGetBatchesByStatusReady() {
+		var ready = READY;
```

Comment:
No need to define this as a local variable - just use the enum constant directly.